andela/octopus-bn-backend
# #171423226 refactor i18n

### What does this PR do?

- grab preferred language from the database
- set ```preferedLang ```claim in JWT token
- add ```setLanguage``` in ```utils/international.js```  method to set the language and render it respectively.
- modify  ```user.validation``` to use ``` setLanguage```
-  add locales for ``` fr``` and ```en```

### How should this be manually tested?

- Just clone the repo
- Run ``` npm i ``` to install necessary dependencies
-  Navigate ```utils/international.js``` to check how the functionality works

### Implementation plan
N/A
### Stories associated with PR

[#171423226](https://www.pivotaltracker.com/story/show/171423226)

### Screenshots if any
N/A
